graylog2-web-interface/src/views/components/searchbar/date-time-picker/TimeRangeAddToQuickListButton.tsx

```diff
@@ -0,0 +1,144 @@
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { useCallback, useMemo, useRef, useState } from 'react';
+import { Position } from 'react-overlays';
+import debounce from 'lodash/debounce';
+import isEqual from 'lodash/isEqual';
+
+import { Button, Input, Popover } from 'components/bootstrap';
+import { Icon, Portal, ModalSubmit } from 'components/common';
+import type { TimeRange, KeywordTimeRange } from 'views/logic/queries/Query';
+import { ConfigurationsActions } from 'stores/configurations/ConfigurationsStore';
+import { ConfigurationType } from 'components/configurations/ConfigurationTypes';
+import useSearchConfiguration from 'hooks/useSearchConfiguration';
+import { onSubmittingTimerange } from 'views/components/TimerangeForForm';
+import useUserDateTime from 'hooks/useUserDateTime';
+import { Link } from 'components/common/router';
+import Routes from 'routing/Routes';
+import type { QuickAccessTimeRange } from 'components/configurations/QuickAccessTimeRangeForm';
+import generateId from 'logic/generateId';
+import useSendTelemetry from 'logic/telemetry/useSendTelemetry';
+
+type Props = {
+  addTimerange: (title: string) => void,
+  toggleModal: () => void,
+  target: Button | undefined | null,
+  equalTimerange: QuickAccessTimeRange
+};
+
+const isTimerangeEqual = (firstTimerange: TimeRange, secondTimerange: TimeRange) => {
+  if (firstTimerange.type !== secondTimerange.type) return false;
+  if (firstTimerange.type === 'keyword') return (firstTimerange as KeywordTimeRange).keyword === (secondTimerange as KeywordTimeRange).keyword;
+
+  return isEqual(firstTimerange, secondTimerange);
+};
+
+const TimeRangeAddToQuickListForm = ({ addTimerange, toggleModal, target, equalTimerange } : Props) => {
+  const [description, setDescription] = useState('');
+  const debounceHandleOnChangeDescription = debounce((value: string) => setDescription(value), 300);
+
+  const onAddTimerange = useCallback(() => addTimerange(description), [addTimerange, description]);
+
+  return (
+    <Portal>
+      <Position placement="left"
+                target={target}>
+        <Popover title="Add to quick access list"
+                 id="add-to-quick-list-popover"
+                 data-app-section="add-to-quick-list-popover_form"
+                 data-event-element="Add to quick list"
+                 data-testid="add-to-quick-list-popover">
+          <Input type="text"
+                 id="add-to-quick-list-description"
+                 placeholder="Add description..."
+                 title="Time range description"
+                 aria-label="Time range description"
+                 defaultValue={description}
+                 onChange={({ target: { value } }) => debounceHandleOnChangeDescription(value)}
```

Comment:
Let's add a margin between the input and the buttons.
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/ef8f943b-c2e2-4da0-bbcd-ed50ce5ed12c)